sdk/confidentialledger/Azure.Security.ConfidentialLedger/tests/ConfidentialLedgerClientLiveTests.cs

```diff
@@ -58,83 +58,83 @@ public async Task GetUser(string objId)
-            while (nextLinkDetails != null)
-            {
-                var fromId = nextLinkDetails["fromTransactionId"];
-                var subId = nextLinkDetails["subLedgerId"];
-                result = await Client.GetLedgerEntriesAsync(subId, fromId).ConfigureAwait(false);
-                Assert.AreEqual((int)HttpStatusCode.OK, result.Status);
-                nextLinkDetails = GetNextLinkDetails(result);
-            }
-        }
-
-        [RecordedTest]
-        public async Task GetLedgerEntry()
-        {
-            await PostLedgerEntry();
-
-            var result = await Client.GetLedgerEntriesAsync();
-            var stringResult = new StreamReader(result.ContentStream).ReadToEnd();
-
-            while (stringResult.Contains("Loading"))
-            {
-                result = await Client.GetLedgerEntriesAsync().ConfigureAwait(false);
-                stringResult = new StreamReader(result.ContentStream).ReadToEnd();
-            }
-            var transactionId = GetFirstTransactionId(result);
-            result = await Client.GetLedgerEntryAsync(transactionId).ConfigureAwait(false);
-
-            Assert.AreEqual((int)HttpStatusCode.OK, result.Status);
-            Assert.That(stringResult, Does.Contain(transactionId));
-        }
-
-        [RecordedTest]
-        public async Task GetReceipt()
-        {
-            await PostLedgerEntry();
-
-            var result = await Client.GetLedgerEntriesAsync();
-            var stringResult = new StreamReader(result.ContentStream).ReadToEnd();
-
-            while (stringResult.Contains("Loading"))
-            {
-                result = await Client.GetLedgerEntriesAsync().ConfigureAwait(false);
-                stringResult = new StreamReader(result.ContentStream).ReadToEnd();
-            }
-            var transactionId = GetFirstTransactionId(result);
-            result = await Client.GetReceiptAsync(transactionId).ConfigureAwait(false);
-
-            Assert.AreEqual((int)HttpStatusCode.OK, result.Status);
-            Assert.That(stringResult, Does.Contain(transactionId));
-        }
-
-        [RecordedTest]
-        public async Task GetTransactionStatus()
-        {
-            await PostLedgerEntry();
-
-            var result = await Client.GetLedgerEntriesAsync();
-            var stringResult = new StreamReader(result.ContentStream).ReadToEnd();
-
-            while (stringResult.Contains("Loading"))
-            {
-                result = await Client.GetLedgerEntriesAsync().ConfigureAwait(false);
-                stringResult = new StreamReader(result.ContentStream).ReadToEnd();
-            }
-            var transactionId = GetFirstTransactionId(result);
-            result = await Client.GetTransactionStatusAsync(transactionId).ConfigureAwait(false);
-
-            Assert.AreEqual((int)HttpStatusCode.OK, result.Status);
-            Assert.That(stringResult, Does.Contain(transactionId));
-        }
+        //[RecordedTest]
```

Comment:
why?

Reply:
These tests have been doing manual iteration over pageable request, now it is handled by codegen. I'm not sure what can be tested here.

Reply:
The tests validate that the API works as expected.